http.go

```diff
@@ -191,32 +206,50 @@ func (router *Router) Bind() {
 
 func fetchSkin(username string) *mcSkin {
 	if username == "char" || username == "MHF_Steve" {
-		skin, _ := minecraft.FetchSkinForChar()
+		skin, _ := minecraft.FetchSkinForSteve()
 		return &mcSkin{Skin: skin}
 	}
 
+	hasTimer := prometheus.NewTimer(cacheDuration.WithLabelValues("has"))
 	if cache.has(strings.ToLower(username)) {
+		hasTimer.ObserveDuration()
+		pullTimer := prometheus.NewTimer(cacheDuration.WithLabelValues("pull"))
+		defer pullTimer.ObserveDuration()
 		stats.HitCache()
 		return &mcSkin{Processed: nil, Skin: cache.pull(strings.ToLower(username))}
 	}
+	hasTimer.ObserveDuration()
 
-	skin, err := minecraft.FetchSkinFromMojang(username)
+	uuid, err := minecraft.NormalizePlayerForUUID(username)
```

Comment:
*this is rate limiting on an empty cache when live*

```
[20:45:17.399568] DEBU Failed UUID lookup: Notch (unable to GetAPIProfile: rate limited)
```

A good few minutes later and still blocked:

```
$ curl https://api.mojang.com/users/profiles/minecraft/LukeHandle
{"error":"TooManyRequestsException","errorMessage":"The client has sent too many requests within a certain amount of time"}
```

Either the initial herd of traffic is too large when trying an empty cache, or I'm missing some caching layer somewhere here. This should only be called when not in the Redis cache, so it's not exactly being hit 24/7. But it is late 😴